.github/workflows/build_and_publish_test_pypi.yml

```
@@ -0,0 +1,149 @@
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.commit }}
+          sparse-checkout: oxen-python/pyproject.toml
+          sparse-checkout-cone-mode: false
+
+      - name: Compute next dev version
+        id: compute
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2001:style:4:11: See if you can use ${variable//search/replace} instead

(shellcheck)